documentation/modules/exploit/multi/php/car_rental_system.md

```diff
@@ -0,0 +1,76 @@
+## Vulnerable Application
+
+Car Rental System 1.0 is a Web Application to rent cars. It was write using PHP and MySQL.
+
+Vendor Homepage: https://www.sourcecodester.com/php/14544/car-rental-management-system-using-phpmysqli-source-code.html
+Software Link: https://www.sourcecodester.com/download-code?nid=14544&title=Car+Rental+Management+System+using+PHP%2FMySQLi+with+Source+Code
+
+to install, extract all files in a web directory, edit admin/db_connect.php, and create database following database/car_rental_db.sql
+
+Opening the web page, we can login in /admin area using " ' OR 1=1 limit 1# " as username, and a random word as password; Giving "limit 1" we will select the first user: ADMIN
```

Comment:
This is way too long of a line, and we are in markdown so backticks will work to highlight things as code which is preferred vs double quotes:

```suggestion
Users can log into the app as an administrator by using the username ` ' OR 1=1 limit 1#` , 
and a random word as password. `limit 1` is used as it ensures the database will select the first
user in the database, ADMIN, thereby logging the attacker in as the administrative user.
```